terra_notebook_utils/cli/drs.py

```diff
@@ -60,7 +60,7 @@ def drs_copy_batch(args: argparse.Namespace):
 @drs_cli.command("head", arguments={
     "drs_url": dict(type=str),
     "--bytes": dict(type=int, required=False, default=1, help="Number of bytes to fetch."),
-    "--buffer": dict(type=int, required=False, default=MULTIPART_THRESHOLD,
+    "--buffer": dict(type=int, required=False, default=None,
```

Comment:
Could the default value (or the description that all data will be loaded without a buffer) be included in the `help`?

Reply:
The size of `buffer` is no longer meaningful.

This should be revisited in a future PR. I will file an issue.